src/libs/UnreadIndicatorUpdater/updateUnread/index.website.js

```diff
@@ -9,7 +9,7 @@ import CONFIG from '../../../CONFIG';
  * @param {Number} totalCount
  */
 function updateUnread(totalCount) {
-    const hasUnread = totalCount > 0;
+    const hasUnread = totalCount !== 0;
```

Comment:
Can't the totalCount still be -1?  Are we going to be showing a NEW message if that is the case?

Reply:
Yep! Since that would mean we have unread messages in group chats (so hence should show `NEW`)